spec/controllers/dashboard_controller_spec.rb

```diff
@@ -2,11 +2,14 @@
 require "rails_helper"
 
 RSpec.describe DashboardController, type: :controller do
-  let(:user)   { create(:user) }
-  let(:minion) { create(:minion) }
+  let(:user)          { create(:user) }
+  let(:minion1)       { create(:minion) }
+  let(:minion2)       { create(:minion) }
+  let(:master_minion) { create(:master_minion) }
+  let(:worker_minion) { create(:worker_minion) }
 
   before do
-    Minion.create! [{ hostname: "master" }, { hostname: "minion0" }]
+    minion1 && minion2
```

Comment:
Would it make sense to simply `let!(:minion1)` and `let(:minion2)` above ? Otherwise doing `minion1 && minion2` is confusing for someone that hasn't done too much rspec.